deebot_client/hardware/deebot/p1jij8.py

```diff
@@ -120,6 +121,17 @@
                     WorkMode.VACUUM_AND_MOP,
                 ),
             ),
+            pads_cleaning_interval=CapabilitySetTypes(
+                event=PadsCleaningIntervalEvent,
+                get=[GetPadsCleaningInterval()],
+                set=SetPadsCleaningInterval,
+                types=(
+                    SetPadsCleaningInterval.DEEP,
```

Comment:
`SetPadsCleaningInterval` doesn't exist. Did you commit and push all the code?

Reply:
I wrote SetPadsCleaningInterval in deebot_client/commands/json/pads_cleaning_interval.py, do I need to add it somewhere else?